src/main/config.js

```diff
@@ -35,3 +35,7 @@ export const VIEW_MENU_ITEM = {
   'Typewriter Mode': false,
   'Focus Mode': false
 }
+
+export const LINE_ENDING_REG = /(\r\n|\n)/g
```

Comment:
is it needed to use a catch group?

```javascript
export const LINE_ENDING_REG = /(?:\r\n|\n)/g
```

Dose bellow code work?

Reply:
Seems to work and should allocate less memory :+1: